### PR TITLE
[merged] SELinux: Don't scan with selinux separation if scan is using rootfs

### PR DESCRIPTION
**Pull request by rhatdan**
We don't have a way to relabel content off of random rootfs so that
a SELinux confined process can read the content.

### CODE REVIEW

**Comment by rhatdan** [comment]
@baude  could you try this patch out, I don't have the environment to test.

File: Atomic/scan.py [hunk 1]
```diff
@@ -77,13 +77,15 @@ def get_scan_info(scanner, scan_type):
...
-
+            security_args = []
         else:
             # Check to make sure all the chroots provided are legit
             for _path in self.args.rootfs:
                 if not os.path.exists(_path):
                     raise ValueError("The path {} does not exist".format(_path))
             self.setup_rootfs_dirs()
+            # Have to disable SELinux checking of scanning not docker images
+            security_args = [ "--security-opt", "label=disable" ]
```
Comment (baude) [inline Atomic/scan.py]: should be label:disable

**Comment by baude** [comment]
With the one correction, it is good.  Feel free to merge

File: Atomic/scan.py (hunk 1 above)
Comment (rhatdan) [inline Atomic/scan.py]: They are changing the default to label=disable

**Comment by rhatdan** [comment]
@baude did you try it out?

**Comment by baude** [comment]
Yeah and it failed because of the label syntax

**Comment by rhatdan** [comment]
It works with the ":",  I guess this is fixed in newer dockers...

**Comment by rhatdan** [comment]
@baude See if it works with the ":"

**Comment by baude** [comment]
Works perfectly for me

**Comment by rhatdan** [comment]
@cgwalters-bot r+ 7326c2d

**Pull request closed without merging**